lib/ansible/modules/cloud/azure/azure_rm_netapp_volume.py

```diff
@@ -0,0 +1,251 @@
+#!/usr/bin/python
+#
+# (c) 2019, NetApp, Inc
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'certified'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_netapp_volume
+
+short_description: Manage NetApp Azure Files Volume
+version_added: "2.9"
+author: NetApp Ansible Team (@carchi8py) <ng-ansibleteam@netapp.com>
```

Comment:
You should move the author to the end of the document and change the format as follows:
```suggestion
author:
    - NetApp Ansible Team (@carchi8py)
```